src/CMakeLists.txt

```diff
@@ -63,6 +63,11 @@ if (MPI_FOUND)
   target_link_libraries(decomp2d PRIVATE MPI::MPI_Fortran)
 endif (MPI_FOUND)
 
+find_package(OpenMP REQUIRED)
```

Comment:
If OpenMP is not available, the code can not compile. I think REQUIRED should be removed here.